conduction-design-tokens/src/component/conduction/card.tokens.json

```diff
@@ -6,6 +6,17 @@
       "border-radius": { "value": "16px" },
       "border-width": { "value": "unset" },
       "color": { "value": "{conduction.color.black.0}" },
+      "header": {

```

Comment:
As discussed in our call: this card component should become a CardWrapper component, that's managed within the `conduction/components` package.